guide/naming.md

```diff
@@ -0,0 +1,58 @@
+# VDD Naming Convetions / Semantic Versioning
```

Comment:
```suggestion
# VDD Naming Conventions / Semantic Versioning
```

Reply:
> Although we will have duplication, we would be able to compare different versions without the need to check out different git branches.

I think I may not be understanding the proposed convention then, cause this part make sit sound like the differently versioned files will be named based on which branch they live on:

```
- There should be an additional stage `Withdrawn`, for the uncommon
  outcome that at some point we find the specification useless
- `Draft` should live in auxiliary branches
- `Proposal` should live in a PR
- `Reviewed`, `Published`, `Withdrawn` should live in the main
  (f.k.a. master) branch
```

if the suggestion is to have different draft versions which live *side by side* on the same branch, that actually makes a lot more sense to me.